app/controllers/v1/legislators_controller.rb

```diff
@@ -0,0 +1,14 @@
+class V1::LegislatorsController < V1::BaseController
+  def index
+    @user = Person.find_or_initialize_by(email: params[:email])
+    if @user.save
+      @user.update_location( address: params[:address],
+                             city:    params[:city],
+                             state:   params[:state],
+                             zip:     params[:zip] )
+    else
+      @error = 'user not found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.